### PR TITLE
chore: maintainer & reviewer changes for CAPA

### PR DESCRIPTION
**Pull request by richardcase**
Updates to reflect recent changes to maintainers of CAPA

### CODE REVIEW

File: k8s.gcr.io/images/k8s-staging-cluster-api-aws/OWNERS [hunk 1]
```diff
@@ -2,15 +2,16 @@
...
+- vincepri
 
 emeritus_approvers:
 - chuckha
 - detiber
 - ncdc
 - randomvariable
 - rudoi
-- vincepri
+- sedefsavas
```
Comment (upodroid) [inline k8s.gcr.io/images/k8s-staging-cluster-api-aws/OWNERS:15]: Hey, can you make this change in the `registry.k8s.io` folder instead? This folder is frozen an no longer accepting changes

Comment (richardcase) [reply k8s.gcr.io/images/k8s-staging-cluster-api-aws/OWNERS:15]: Of course, sorry about that.

Comment (richardcase) [reply k8s.gcr.io/images/k8s-staging-cluster-api-aws/OWNERS]: Updated to use registry.k8s.io

**Comment by richardcase** [comment]
@fabriziopandini - if you get time could you look at this from a SCL point of view?

**Review by neolit123 — COMMENTED** [review]
/assign @vincepri 
i have no context on these changes.
generally i see a lot if inactive folks in these lists

File: groups/sig-cluster-lifecycle/groups.yaml [hunk 2]
```diff
@@ -264,7 +264,6 @@ groups:
       - richmcase@gmail.com
       - sedefsavas@gmail.com
```
Comment (sbueringer) [inline groups/sig-cluster-lifecycle/groups.yaml:265]:
```suggestion
```

**Comment by fabriziopandini** [comment]
+1 from my side pending the nit above

**Pull request merged**